config/settings.yml

```diff
@@ -401,6 +401,14 @@ oidc:
 sentry:
   dsn: ~
 
+iam_ssoe:
+  client_cert_path: /fake/client/cert/path
+  client_key_path: /fake/client/key/path
```

Comment:
Assume these will be filled in later

Reply:
This settings file gets overridden by the dev, stage, prod ones. Although since the key and cert get mocked out in specs it might be better to have these be nil.